ui/src/components/Toolbar/PagingButtons.jsx

```diff
@@ -64,6 +65,42 @@ class PagingButtons extends React.Component {
     });
   };
 
+  renderFeedback() {
+    const { document } = this.props;
+
+    const popoverContent = (
+      <>
+        <p>
+          <strong>
+            <FormattedMessage
+              id="document.report_problem.title"
+              defaultMessage="Does the document preview not look quite right? Or is the extracted text incomplete?"
+            />
+          </strong>
+        </p>
+        <p>
+          <FormattedMessage
+            id="document.report_problem.text"
+            defaultMessage="You can now easily report such problems to the Aleph team. This helps us improve how Aleph processes and displays documents."
+          />
+        </p>
+      </>
+    );
+
+    return (
+      <HintPopover id="document-feedback" content={popoverContent}>
```

Comment:
Note to myself: This will display the hint popover even if the feedback URL is not configured.